### PR TITLE
Tf rawops cleanup

### PR DESCRIPTION
**Pull request by YushaArif99**
previously made an incorrect PR but nevertheless committed the changes to the master. This new PR further replaces 2 more function calls in the raw_ops.py file with their corresponding aliases.

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def Acosh(*, x, name="Acosh"):
 
 
 def Add(*, x, y, name="Add"):
-    return ivy.add(x, y)
+    return tf_frontend.add(x, y)
```
Comment (djl11) [inline ivy/functional/frontends/tensorflow/raw_ops.py:20]: if it's an alias, we should simply write `Add = tf_frontend.add` :)

File: .idea/vcs.xml [hunk 2]
```diff
@@ -2,5 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ivy_tests/test_array_api" vcs="Git" />
```
Comment (djl11) [inline .idea/vcs.xml:5]: please remove `.idea` changes from the commit

**Review by djl11 — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 1 above)
Comment (YushaArif99) [reply ivy/functional/frontends/tensorflow/raw_ops.py]: Oh my bad. I'll make the required changes

File: .idea/vcs.xml (hunk 2 above)
Comment (YushaArif99) [reply .idea/vcs.xml]: Didn't mean to add this file. Must have been added accidentally when I committed the changes. Nonetheless, I'll rollback the changes

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 3]
```diff
@@ -17,11 +17,13 @@ def Acosh(*, x, name="Acosh"):
 
 
 def Add(*, x, y, name="Add"):
-    return ivy.add(x, y)
+    Add = tf_frontend.add
```
Comment (djl11) [inline ivy/functional/frontends/tensorflow/raw_ops.py:20]: no you can remove the entire function definition, and just write `Add = tf_frontend.add` as one line in this file, nothing else is needed :)

**Review by djl11 — CHANGES_REQUESTED** [review]

**Comment by djl11** [comment]
the line ending in one of the `.idea` files has also changed, please remove all `.idea` changes from the commit. Might be easiest to `rebase` the fork and then `push --force` or something

**Comment by YushaArif99** [comment]
I have also updated the description in the `Add RawOps Operations for TensorFlow frontend` ToDo list (https://github.com/unifyai/ivy/issues/1565). Could you also check it out and let me know if the description needs any changes

**Review by djl11 — APPROVED** [review]
lgtm!

**Pull request merged**